src/app/components/header/header.component.scss

```diff
@@ -2,22 +2,25 @@
 
 .header {
     height: 60px;
-    font-size: 12px;
-    line-height: 14px;
+    font-size: 14px;
+    line-height: 16px;
 
     &--mobile {
         height: unset;
     }
 
     &__balance {
-        text-transform: uppercase;
+        color: $light-grey;
+
+        &--eth {
```

Comment:
Nitpick: modifiers of the BEM scheme are usually behavior descriptive adjectives. So eventually `--upper-case` or similar fits better. This would also make it re-usable in case you would need it for anything else that needs the same modification. But this is really a stupid mistake. :laughing:

Reply:
You are totally right :)